### DISCUSSION TITLE
zap an issue as a bounty?

### DISCUSSION THREAD

**Discussion by PeerRich** [NostrGit/NostrGit#102] [General]
could we build it in a way that you zap an issue (bounty) but instead the sats go to the author of the PR that closes it?

it would need a temp LN wallet that then sends total amount to the author of the PR

that way people can zap an issue to give it more priority

**Comment by cypherhoodlum** [comment]
Good idea, I like the concept. :+1: The wallet management should probably be highly automated.

**Reply by alltheseas** [reply]
> could we build it in a way that you zap an issue (bounty) but instead the sats go to the author of the PR that closes it?

> it would need a temp LN wallet that then sends total amount to the author of the PR

> that way people can zap an issue to give it more priority

See related request & mockup https://github.com/NostrGit/NostrGit/issues/103.

> Good idea, I like the concept. 👍 The wallet management should probably be highly automated.

How are sats split between PRs where more than one dev participated? 

For instance tickets where there were:
1) a main dev, and a reviewer (e.g. 90/10)?
2) at least two devs, who contributed a similar amount (e.g. 50/50)?

What if there was a blocking ticket, technical or non-technical that had to be done first, prior to completing the bounty ticket?

Who decides the contribution levels?

**Comment by pierresisson** [comment]
upvoted!

**Comment by cypherhoodlum** [comment]
Let's continue the discussion at #103 :heart:
